### ISSUE TITLE
DeprecationWarning error

### ISSUE THREAD

**Issue by varunagrawal** [jdan/cleaver#171]
When I run `cleaver test.md` I get the following error:

```shell
(node:2576) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
```

Here is the debug trace:

```shell
cleaver loaded input document +0ms
  helper read /home/varun/.linuxbrew/lib/node_modules/cleaver/templates/layout.mustache +2ms
  helper read /home/varun/.linuxbrew/lib/node_modules/cleaver/templates/default.mustache +0ms
  helper read /home/varun/.linuxbrew/lib/node_modules/cleaver/resources/default.css +0ms
  helper read /home/varun/.linuxbrew/lib/node_modules/cleaver/resources/github.css +0ms
  helper read /home/varun/.linuxbrew/lib/node_modules/cleaver/templates/author.mustache +0ms
  cleaver loaded templates +0ms
  cleaver parsed options +3ms
  helper read /home/varun/.linuxbrew/lib/node_modules/cleaver/resources/script.js +4ms
  cleaver loaded static assets +0ms
  cleaver rendered slides +1ms
  cleaver rendered presentation +0ms
(node:2576) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
```

The `test.md` is the first example in the Cleaver README.

**Comment by sudodoki** [comment]
seems to be coming from [this](https://github.com/jdan/cleaver/blob/master/bin/cleaver#L15-L38) and the fact that it's called sometimes with cb, sometimes without. Can leverage default arguments and do `noop` or pass it explicitly everywhere, where done is not passed.

**Comment by juanbrujo** [comment]
😱  having the same error!! can this be solved soon?

**Comment by sudodoki** [comment]
Fixed via #172 and published in https://github.com/jdan/cleaver/releases/tag/v0.8.5
